lib/snapshot/BUILD.gn

```diff
@@ -198,12 +198,18 @@ if (host_os == "mac" && (target_os == "mac" || target_os == "ios")) {
     gen_snapshot_target =
         "$dart_src/runtime/bin:$gen_snapshot_target_name($build_toolchain)"
 
-    copy(target_name) {
+    action(target_name) {
```

Comment:
What's the rationale for only using it here and not updating the toolchain definition so that all copy targets use it?

Reply:
It was to limit impact to only `gen_snapshot`s. Only them had an issue with Gatekeeper.
Generally speaking `ln` has an advantage of reduced disk space usage.

Reply:
Presumably, Gatekeeper would also impact other binaries as well. If this is the fix for `gen_snapshot`, then we wouldn't want to go through all this investigation again a second time for some other binary. I think it's fine to do it just for `gen_snpashot` for now as an experiment, but if it works, there has to be follow-up work to deploy the solution more generally.